libs/core/langchain_core/messages/tool.py

```diff
@@ -61,3 +63,105 @@ def __add__(self, other: Any) -> BaseMessageChunk:  # type: ignore
             )
 
         return super().__add__(other)
+
+
+class ToolCall(Serializable):
```

Comment:
How will this get serialized? `langchain.messages` or `langchain.messages.tool`?